src/ir_Mitsubishi.cpp

```diff
@@ -1011,19 +998,17 @@ void IRMitsubishi136::setSwingV(const uint8_t position) {
     case kMitsubishi136SwingVHigh:
     case kMitsubishi136SwingVHighest:
     case kMitsubishi136SwingVAuto:
-      setBits(&remote_state[kMitsubishi136SwingVByte], kHighNibble, kNibbleSize,
-              position);
+      _.SwingV = position;
       break;
     default:
-      setMode(kMitsubishi136SwingVAuto);
```

Comment:
is this a bug?

Reply:
I think so.
Looks like it should be:
`setSwingV(kMitsubishi136SwingVAuto);`

Can you fix these in a separate PR please? i.e. Keeps this PR just the re-factor, and we can list the subsequent PR in the [bug] section in the release notes for the next release.